content/tools.md

```diff
@@ -5,9 +5,24 @@ toc: true
 
 ## Science
 
+### General Tools
+
 - [Animations of the free oscillations of the Earth](https://saviot.cnrs.fr/terre/index.en.html)
+- [Google Maps API](https://developers.google.com/maps/get-started/)
 - [Interactive periodic table of the elements](https://leonard-seydoux.github.io/periodic-table/)
 - [Periodic Table of Elements](https://www.ptable.com)
+- [3D earthquake display](https://glowy-earthquakes.glitch.me)
+
+### IRIS
+
+- [IRIS-DMC MetaData Aggregator](http://ds.iris.edu/mda2/): View information of all stations
+- [IRIS Data Request Status](http://www.iris.edu/ds/nodes/dmc/data/request-status/)
+- [IRIS GMAP](http://ds.iris.edu/gmap/): View stations on Google Maps
+- [IRIS Message Center](http://ds.iris.edu/message-center/)
+- [IRIS Seismic Monitor](http://ds.iris.edu/seismon/index.phtml)
+- [IRIS Station Monitor](https://www.iris.edu/app/station_monitor/)
+- [IRIS Synthetics Engine](http://service.iris.edu/irisws/syngine/1/)
+- [Wilber 3](http://ds.iris.edu/wilber3/): Request waveform of single event
```

Comment:
Already in the "Codes" category.
```suggestion
```

Reply:
I don't see IRIS Synthetics Engine in codes.

Reply:
IRIS Synthetic Engine is removed in 38ac9e7. I found it is [Instaseis](http://instaseis.net).